### PR TITLE
Shadow update

### PR DESCRIPTION
**Pull request by shadetree01010100**

### CODE REVIEW

File: aws_iot_update_shadow_block.py [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    """Update a device's shadow document with a reported state."""
+
+    version = VersionProperty('1.0.0')
+    thing_name = StringProperty(title='Thing Name')
+    reported_state = Property(
+        title='Reported State', default='{{ $.to_dict() }}')
+
+    def __init__(self):
+        super().__init__()
+        self.client = deepcopy(AWSIoTMQTTShadowClient)
```
Comment (mattdodge) [inline aws_iot_update_shadow_block.py]: What's this `deepcopy` about? Seems bizarre

**Review by mattdodge — CHANGES_REQUESTED** [review]
Nice! This looks great. One questionable deepcopy but otherwise this is good to go

File: aws_iot_update_shadow_block.py (hunk 1 above)
Comment (shadetree01010100) [reply aws_iot_update_shadow_block.py]: I was simply following the pattern in the base's `__init__`
https://github.com/nio-blocks/AWS_IoT/blob/shadow_update/aws_iot_mqtt_base_block.py#L40

Looking at history now to try and figure out why.

Comment (mattdodge) [reply aws_iot_update_shadow_block.py]: Ok yeah, that seems very strange. See if you can track it down and maybe what happens if we get rid of it. It's possible it was there to "test" something and never got removed. Can't imagine why we'd want to deepcopy a 3rd party library's class

**Review by mattdodge — APPROVED** [review]

**Pull request merged**